plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeMetadata.java

```diff
@@ -313,32 +313,6 @@ public void testGetInsertLayoutTableUnpartitioned()
                 .isNotPresent();
     }
 
-    @Test
-    public void testGetInsertLayoutTableNotFound()
-    {
-        SchemaTableName schemaTableName = newMockSchemaTableName();
-
-        DeltaLakeTableHandle missingTableHandle = new DeltaLakeTableHandle(
-                schemaTableName.getSchemaName(),
-                schemaTableName.getTableName(),
-                getTableLocation(schemaTableName),
-                Optional.empty(),
-                TupleDomain.none(),
-                TupleDomain.none(),
-                Optional.empty(),
-                Optional.empty(),
-                Optional.empty(),
-                Optional.empty(),
-                Optional.empty(),
-                0,
-                false);
-
-        assertThatThrownBy(() -> deltaLakeMetadataFactory.create(SESSION.getIdentity())
-                .getInsertLayout(SESSION, missingTableHandle))
```

Comment:
Note: `testGetInsertLayoutTableNotFound` test was removed, instead of
being updated, since `ConnectorMetadata.getInsertLayout` cannot be
reached for a corrupted table, as getting column list will fail earlier.